### PR TITLE
Update requirements.txt for SQLAlchemy

### PR DESCRIPTION
**Pull request by Netmisa**
Screenshot from 2019-04-17 10-53-27

### CODE REVIEW

**Comment by Netmisa** [comment]
```
[2019-04-17 11:03:13,650] [ERROR] [26314] [            chaos.navitia] call to navitia timeout
[2019-04-17 11:03:13,751] [ERROR] [26314] [            chaos.navitia] call to navitia timeout
[2019-04-17 11:03:13,851] [ERROR] [26314] [            chaos.navitia] call to navitia timeout
```

**Comment by Netmisa** [comment]
Jenkins could you test this please.

**Comment by pthegner** [comment]
retest this please

**Comment by Msaglier** [comment]
Error 500 on POST /impacts/_search

**Comment by pthegner** [comment]
retest this please

**Comment by Fahmus** [comment]
jenkins, retest this please

File: requirements.txt [hunk 1]
```diff
@@ -30,7 +30,7 @@ redis==2.9.1
 requests==2.20.0
 retrying==1.3.3
 six==1.11.0
-SQLAlchemy==0.9.7
+SQLAlchemy==1.3.0
```
Comment (dvdn) [inline requirements.txt]: Maybe consider a smaller step, before big jump to 1.3
```suggestion
SQLAlchemy==1.1.18
```

**Comment by Netmisa** [comment]
See together (team)
- This PR need more refactoring to upgrade `SQLAlchemy==1.3.0` and fix it.
- It's not a priority because we are not using dynamic `group_by` and `order_by`

We will postpone this fix.

**Pull request closed without merging**